### DISCUSSION TITLE
Feature request: 2D & 3D denoising methods

### DISCUSSION THREAD

**Discussion by alexjnoble** [nextpyp/.github#19] [Ideas]
Hello,

It would be useful to optionally denoise micrographs and tomograms. There are several software packages out there that will do this. I am familiar with Topaz, which has general 2D and 3D denoising models, so training is not necessary. Could this be added to the nextPYP workflows?

Thanks,
-Alex

**Comment by abartesaghi** [comment]
We haven't found the need for 2D/3D denoising in our workflows, but it should not be difficult to integrate tools like Topaz denoise into nextPYP if users find it useful.

**Reply by abartesaghi** [reply]
Support for Topaz denoise 3D using general models is available starting with version [0.6.0](https://nextpyp.app/files/pyp/0.6.0/docs/index.html).
